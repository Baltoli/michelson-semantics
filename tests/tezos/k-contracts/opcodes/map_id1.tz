$contract { parameter (map nat nat);
storage (map nat nat);
code { CAR ; NIL operation ; PAIR }
 } ;
$param { Elt 0 0 } ;
$storage {} ;
$ExpectedOutput { Elt 0 0 } ;

