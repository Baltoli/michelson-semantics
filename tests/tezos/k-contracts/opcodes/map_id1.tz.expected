<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List #Map ( 0 |-> 0 , nat .AnnotationList , nat .AnnotationList )
  </stack>
  <paramtype>
    map .AnnotationList nat .AnnotationList nat .AnnotationList
  </paramtype>
  <storagetype>
    map .AnnotationList nat .AnnotationList nat .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
